packages/shared.test/src/stubs/actions.ts

```diff
@@ -299,115 +179,74 @@ const actionResponse: SyncApplyActionResponseV2 = {
-  validation: {
-    result: "INVALID",
-    submissionCriteria: [],
-    parameters: {},
-  },
-};
-
-const actionResponseMoveOfficeBatch: BatchApplyActionResponseV2 = {
-  edits: {
-    type: "edits",
-    edits: [{
-      type: "modifyObject",
-      primaryKey: "SEA",
-      objectType: officeObjectType.apiName,
-    }, {
-      type: "modifyObject",
-      primaryKey: "NYC",
-      objectType: officeObjectType.apiName,
-    }],
-    addedObjectCount: 0,
-    addedLinksCount: 0,
-    modifiedObjectsCount: 2,
-    deletedLinksCount: 0,
-    deletedObjectsCount: 0,
-  },
-};
-
-export const actionResponseMap: {
-  [actionApiName: string]: {
-    [actionBody: string]: any;
-  };
-} = {
-  promoteEmployee: {},
-  createOffice: {
-    [stableStringify(actionRequestCreateOffice)]: actionResponseCreateOffice,
-    [stableStringify(actionRequestCreateOfficeNoReturnEdits)]: actionResponse,
-  },
-  moveOffice: {
-    [stableStringify(actionRequestMoveOffice)]: actionResponse,
-    [stableStringify(actionRequestMoveOfficeBig)]: actionResponseMoveOfficeBig,
-    [stableStringify(actionRequestMoveOfficeGetResults)]:
-      actionResponseGetResults,
-    [stableStringify(actionRequestMoveOfficeValidateOnly)]:
-      actionResponseInvalid,
-    [stableStringify(actionRequestMoveOfficeValidateOnlyWithoutEdits)]:
-      actionResponseInvalid,
-    [stableStringify(actionRequestMoveOfficeInvalid)]: actionResponseInvalid,
-    [stableStringify(actionRequestMoveOffice2)]: undefined,
-    [stableStringify(actionRequestMoveOffice3)]: undefined,
-    [stableStringify(actionRequestMoveOfficeBatch)]: {},
-    [stableStringify(actionRequestMoveOfficeBatchWithEdits)]:
-      actionResponseMoveOfficeBatch,
-    [stableStringify(actionRequestMoveOfficeBatchInvalid)]: undefined,
-  },
-  createOfficeAndEmployee: {
-    [stableStringify(actionRequestCreateOfficeAndEmployee)]:
-      actionResponseCreateOfficeAndEmployee,
-  },
-  actionTakesObjectSet: {
-    [stableStringify(actionRequestWithObjectSet)]: actionResponse,
-  },
-  actionTakesAttachment: {
-    [stableStringify(actionRequestWithAttachment)]: actionResponse,
-    [stableStringify(actionRequestWithAttachmentUpload)]: actionResponse,
-  },
-  actionTakesMedia: {
-    [stableStringify(actionRequestMediaUpload)]: actionResponse,
-  },
-  deleteFooInterface: {
-    [stableStringify(actionRequestWithInterface)]: actionResponse,
-  },
-  createStructPerson: {
-    [stableStringify(actionRequestWithStruct)]: actionResponse,
-  },
-  createFooInterface: {
-    [stableStringify(actionRequestWithObjectTypeReference)]: actionResponse,
-  },
-};
+export function registerLazyActions(fauxOntology: FauxOntology): void {
+  // These first actions don't didn't have implementations in the legacy ontology
+  fauxOntology.registerActionType(PromoteEmployee);
+  fauxOntology.registerActionType(ActionTakesAnotherInterface);
+  fauxOntology.registerActionType(PromoteEmployeeObject);
+
+  // junk implementations!
```

Comment:
Damn. I was hoping this would line up with the above so it would be easier to see that little changed :/